src/renderer/components/cluster-manager/bottom-bar.scss

```diff
@@ -17,4 +17,15 @@
       cursor: pointer;
     }
   }
+
+  .extensions {
+    font-size: var(--font-size-small);
+    color: white;
+    .item {
+      &:hover {
+        background-color: #ffffff33;
```

Comment:
This should be part of the colour scheme file instead of being hard coded.

Reply:
This colour was already hardcoded here. I think the problem is that bottom-bar does not follow dark/light theme -> we cannot use those variables here. /cc @aleksfront